doc/lightning_kokkos/device.rst

```diff
@@ -114,4 +114,92 @@ Supported operations and observables
 
 .. raw:: html
 
-    </div>
\ No newline at end of file
+    </div>
+
+**Distributed simulation with MPI:**
+
+The ``lightning.kokkos`` device supports distributed simulation using the Message Passing Interface (MPI). This enables the simulation of larger quantum circuits by distributing the workload across multiple CPU or GPU compute nodes.
+
+To utilize distributed simulation, ``lightning.kokkos`` must be compiled with MPI support. Check out the :ref:`Lightning Kokkos installation guide<install-lightning-kokkos-with-mpi>` for more information.
+
+With ``lightning.kokkos`` installed with MPI support, distributed simulation can be enabled in Pennylane by setting the ``mpi`` keyword argument to ``True`` when creating the device. For example:
+
+.. code-block:: python
+
+    from mpi4py import MPI
+    import pennylane as qml
+
+    dev = qml.device('lightning.kokkos', wires=8, mpi=True)
+    @qml.qnode(dev)
+    def circuit_mpi():
+        qml.PauliX(wires=[0])
+        return qml.state()
+    local_state_vector = circuit_mpi()
+
+.. note::
+    The total number of MPI processes must be powers of 2. For example, 2, 4, 8, 16, etc. If using Kokkos with GPUs, we recommend using one GPU per MPI process.
```

Comment:
minor: I found the example too redundant. feel free to ignore this comment if you want to keep it though.
```suggestion
    The total number of MPI processes must be powers of 2. If using Kokkos with GPUs, we recommend using one GPU per MPI process.
```